### ISSUE TITLE
Missing Task should fail linter

### ISSUE THREAD

**Issue by admiralAwkbar** [IBM/tekton-lint#20]
Hey All!

Im trying to add the `Tekton-lint` to the **GitHub Super-Linter** and have 1 ask for improvement.

Currently, if you run the following code:
```yml
apiVersion: tekton.dev/v1beta1
kind: Task
metadata:
  name: example-task-name
spec:
  params:
    - name: pathToDockerFile
      type: string
      description: The path to the dockerfile to build
      default: /workspace/workspace/Dockerfile
  resources:
    inputs:
      - name: workspace
        type: git
    outputs:
      - name: builtImage
        type: image
  steps:
    - name: ubuntu-example
      image: ubuntu
      args: ["ubuntu-build-example", "SECRETS-example.md"]
    - image: gcr.io/example-builders/build-example
      command: ["echo"]
      args: ["$(params.pathToDockerFile)"]
    - name: dockerfile-pushexample
      image: gcr.io/example-builders/push-example
      args: ["push", "$(resources.outputs.builtImage.url)"]
      volumeMounts:
        - name: example-volume
          mountPath: /var/run/docker.sock
  volumes:
    - name: example-volume
      emptyDir: {}
```

It will complete successfully with error code `0`

**But** if i remove the line:
```yml
kind: Task
```
It will run and show no output, and exit with exit code `0`... Can this be set to exit with error code if it fails to read the data?

thanks!

**Comment by Besttwotwofour** [comment]
มันจะทํางานและแสดงไม่มีการส่งออกและออกด้วยรหัสออก 0... สามารถตั้งค่าให้ออกด้วยรหัสข้อผิดพลาดถ้ามันล้มเหลวในการอ่านข้อมูลหรือไม่

ขอบคุณ!

**Comment by madbence** [comment]
👍 I agree @admiralAwkbar, it'd make sense to exit with 0 if none of the input files contained tekton definitions.

**Comment by markkovari** [comment]
@madbence you mentioned above 
> it'd make sense to exit with 0 if none of the input files contained tekton definitions

the [pr](#27) implements kinda the opposite exit code-wise. Should we handle the cases differently when:
 - the linter has been reading some files under the given path, but no tekton-definitions are present 
 - the linter has been trying to read files based on the paths, but there are no such file(s)/path with the given blobs

otherwise if it is implements the requested changes @admiralAwkbar, can we close this one?

**Comment by admiralAwkbar** [comment]
@markkovari I think this shoudl work!

**Comment by madbence** [comment]
@admiralAwkbar I published [`tekton-lint@0.5.1`](https://www.npmjs.com/package/tekton-lint/v/0.5.1) which contains this change!
